tests/foreman/cli/test_computeresource.py

```diff
@@ -31,6 +31,8 @@
 from robottelo.common.constants import FOREMAN_PROVIDERS
 from robottelo.test import CLITestCase
 
+import random
```

Comment:
`import` statements lines should come before `from ... import ...` separated by one line.